src/views/apps.js

```diff
@@ -224,6 +225,11 @@ export const Apps = () => {
   const [instances, setInstance] = useState();
   const [tab, setTab] = useState('Available');
   const [refresh, setRefresh] = useState(false);
+  const [modalOpen, setModalOpen] = useState(false);
```

Comment:
@connectbo how does this fit in with your ongoing changes?

Reply:
It might not be able to fit in directly, since the apps page now is not responsible for rendering specific elements, and each available / active will take care of all the logic. I will take a look and check those out, can make some rearrangement/adjustments if needed.